### PR TITLE
docs: table page for drizzle studio usability features

### PR DESCRIPTION
**Pull request by bgrenon**
Preview:
https://neon-next-git-bgrenon-drizzle-for-sidebar-neondatabase.vercel.app/docs/manage/tables

Relates to [13746](https://github.com/neondatabase/cloud/issues/13746)

### CODE REVIEW

File: content/docs/manage/tables.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Add records
+
+Add new rows to your tables using the **Add record** button.
+
+![add record to table](/docs/manage/add_record_drizzle.png)
+
+A couple of things to note:
+
+- You need to hit `Enter` for your input to register. When editing existing fields, you don't have to do this. But for new fields, if you tab to the next cell, you'll lose your input.
+- You can leave `DEFAUlT` fields untouched and the cell will inherit the right value based on your schema definition. For example, defaults for boolean fields are automatically applied when you click `Save changes`.
```
Comment (danieltprice) [inline content/docs/manage/tables.md:24]:
```suggestion
- You can leave `DEFAULT` fields untouched and the cell will inherit the right value based on your schema definition. For example, defaults for boolean fields are automatically applied when you click `Save changes`.
```

**Review by danieltprice — APPROVED** [review]
Nice gifs!

File: content/docs/manage/tables.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The **Tables** page in the Neon Console offers a dynamic, visual interface for managing data. Fully interactive, this view lets you add, update, and delete records, filter data, modify columns, drop or truncate tables, as well as export data in both .json and .csv formats.
+
+## Edit records
+
+Edit individual entries directly within the table interface. Click on a cell to modify its contents. You don't have to press `Enter` (though you can). Just move your cursor to the next cell you want to modify. Click `Save x changes` when you're done.
+
+![edit table records](/docs/manage/edit_record_drizzle.png)
+
+## Add records
```
[the remark this answers is not in the document]
Comment (bgrenon) [reply content/docs/manage/tables.md:15]: The UI used "record" in button text. I thought I'd use "row" in the text to sort of cover the bases. But if it's confusing, I can consolidate on one.

[the remark this answers is not in the document]
Comment (bgrenon) [reply content/docs/manage/tables.md]: Changed all instances to record. We can request a change to row from Drizzle later.

**Review by russelldias98 — APPROVED** [review]

**Pull request merged**